Transfer/Sources/ScheduleTransferSectionData.swift

```diff
@@ -113,7 +107,7 @@ struct ScheduleTransferSummaryData: ScheduleTransferSectionData {
             rows.append((title: "transfer_fee_confirmation".localized(),
                          value: feeAmount.format(with: destinationCurrency)))
```

Comment:
why fee amount is still being formatted? remove `format(with: destinationCurrency)` from feeAmount